src/wp-includes/comment.php

```diff
@@ -2756,7 +2756,7 @@ function weblog_ping($server = '', $path = '') {
 	// using a timeout of 3 seconds should be enough to cover slow servers
 	$client = new WP_HTTP_IXR_Client($server, ((!strlen(trim($path)) || ('/' == $path)) ? false : $path));
 	$client->timeout = 3;
-	$client->useragent .= ' -- WordPress/' . get_bloginfo( 'version' );
+	$client->useragent .= ' -- ClassicPress/' . get_bloginfo( 'version' );
```

Comment:
We will need to change a lot of these `get_bloginfo( 'version' )` calls to `get_bloginfo( 'classicpress_version'` )` or `classicpress_version()`.  See:  https://github.com/ClassicPress/ClassicPress/pull/32#discussion_r215516465

I think it's fine to do this in a separate PR.